devito/logger.py

```diff
@@ -104,6 +104,14 @@ def set_log_noperf():
     logger.setLevel(WARNING)
 
 
+def would_emit(level):
```

Comment:
This function name seems like "the odd one out" for me, but maybe I am just nitpicking.

Reply:
feels OK to me . it reads quite well "if would_emit(a_certain_log_level) then do something". No?

Reply:
Feels like a repetition of `isEnableFor`